packages/import-mistica-icons/index.js

```diff
@@ -174,8 +174,12 @@ const createIconComponentSource = async (name, componentName, svgIconsInfo) => {
      * To update, execute "yarn start" inside "import-mistica-icons"
      */
 
-    import * as React from 'react';
-    ${hasVariants ? "import {useTheme} from '../../hooks';" : ''}
```

Comment:
this was creating an empty line between imports if `hasVariants` was false, and ESLint was complaining about it